cookbook-react/recipes/tab-bars/tab-bar-1/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import cn from 'classnames';
+
+import './styles.scss';
+
+interface Option {
+  label: string;
+  id: number
+}
+
+interface Props {
+  options?: Option[];
```

Comment:
If this is the only prop and the whole feature depends on it, shouldn't it be a required instead of an optional one? :thinking: